### ISSUE TITLE
PHP 8.1: Memcached::setMulti() throws invalid type error for $expiration

### ISSUE THREAD

**Issue by othercorey** [php-memcached-dev/php-memcached#484]
I created a php bug initially - https://bugs.php.net/bug.php?id=81063.

```
TypeError: Memcached::setMulti(): Argument #2 ($expiration) must be of type int, int given
```

Even when passing an int correctly, it throws an error expecting an int. I don't know if this is related to the implementation, stubs or PHP core. However, no other methods are throwing this error.

**Comment by othercorey** [comment]
@krakjoe With the long time since last release and lack of support > php 7.4, do you know if a php-memcached release is planned? I don't know when/how we'll be able to test with this.

**Comment by Krakozaber** [comment]
Memcached::setMulti(): Argument #2 ($expiration) must be of type int, int given - php8.1

**Comment by LordSimal** [comment]
Would be awesome if @krakjoe or any of the php-memcached-dev team could plan to release a new version including this fix in the near future so PHP 8.1 can properly utilize memcache 😄

**Comment by Sedaj** [comment]
In case somebody stumbles upon this issue, you can install newer version of php memcached extension from ondrej repositoy

`sudo add-apt-repository ppa:ondrej/php`

`sudo apt update`

`sudo apt-get install --only-upgrade php8.1-memcached`

version 3.1.5 gave this error message, but version 3.2.0 was ok for me
